tsconfig.json

```diff
@@ -16,6 +16,11 @@
 		"jsxImportSource": "@emotion/react",
 		"incremental": true
 	},
-	"include": ["**/*.ts", "**/*.tsx", ".storybook/**/*.tsx"],
+	"include": [
+		"**/*.ts",
+		"**/*.tsx",
+		".storybook/**/*.tsx",
+		".storybook/preview.tsx"
```

Comment:
This line isn't covered by the one above?

Reply:
- [x] Remove preview glob